pkg/update/required.go

```diff
@@ -64,9 +64,14 @@ func getRequiredAirgapUpdates(airgap *kotsv1beta1.Airgap, license *kotsv1beta1.L
 		for _, appVersion := range installedVersions {
 			requiredSemver, requiredSemverErr := semver.ParseTolerant(requiredRelease.VersionLabel)
 
+			licenseChan, err := kotsutil.FindChannelInLicense(appVersion.ChannelID, license)
```

Comment:
this should use the app channel id, not the app version channel id. that's because the app version one would be old. we should always use the most recent channel id, that previously came from the license, but with this PR, it should come from the app like everywhere else.

Reply:
Update in Commit: https://github.com/replicatedhq/kots/pull/4767/commits/45ababaa752aae2fe08ccd71a64a7748d056a4a3